genallocs/gen_qi_alloc_cyprus1.json

```diff
@@ -0,0 +1 @@
x00fA49ab81ace6e2958299d0BD9baAa80E4173D4":{"balance":"10000000000000000000000"},"0x00C1de6664584E8eb95AC3Dd8f3C225c4e9f3a1C":{"balance":"10000000000000000000000"},"0x0098D6622D8742470Ad2B73E7fF2983F9e12cac0":{"balance":"10000000000000000000000"},"0x009288FA367D79198D984BB755Fd8Eb7e1AA5496":{"balance":"10000000000000000000000"},"0x00C16fefC14D0d082e701aEBBae79d9c9bD6820b":{"balance":"10000000000000000000000"},"0x0084c1a9E10f9856B6597D81e568f2A63a7210c1":{"balance":"10000000000000000000000"},"0x00b60eD453EA2c9069f34d608b04E6388f1E58D0":{"balance":"10000000000000000000000"},"0x008e3a8b97d1483655942ED3132D2821B92E68Cb":{"balance":"10000000000000000000000"},"0x00b000379521347F914A8F11408263633d06960D":{"balance":"10000000000000000000000"},"0x00c8cb58092ba798c350a980fdD3935c79Dc94A8":{"balance":"10000000000000000000000"},"0x00A2F4dDA0CB85d6ECD30B2c30201e16420B9A3e":{"balance":"10000000000000000000000"},"0x00820Fb8AB3a507a73AcE298aEA1D46F7D8a04aD":{"balance":"10000000000000000000000"},"0x00eb1Fdd32615347F334Bb586ac8FDB282CAdBD1":{"balance":"10000000000000000000000"},"0x00FBDC00896530399A6810F5e2B1626bd2b2A51B":{"balance":"10000000000000000000000"},"0x00cD17fB806474d424aa98420d6c546acC34F1f8":{"balance":"10000000000000000000000"},"0x00dF83E3390Ff851B76d8a298c651e749B40B631":{"balance":"10000000000000000000000"},"0x00aC9f73E68B4A4e1Ea112a7a62f0731dc69e2a0":{"balance":"10000000000000000000000"},"0x00E4d6108f9E41667f0451926873FAf157Cb8aa3":{"balance":"10000000000000000000000"},"0x00d4bA54beCc149ae26AC50F428650fe9d8ccf26":{"balance":"10000000000000000000000"},"0x00a45Bfc12c490A64959ab742815b51b7F7F2AB7":{"balance":"10000000000000000000000"},"0x00B51BE382b67514Af0599438466bF1B861A6C6C":{"balance":"10000000000000000000000"},"0x00f4690443660d7f9AD83861dF0E361804336a0D":{"balance":"10000000000000000000000"},"0x0097544c43dE46BF2DC0a67E9D6ab54864c72f63":{"balance":"10000000000000000000000"},"0x00eD83e2f8C58F9d77101EF35afa8f0b720172d2":{"balance":"10000000000000000000000"},"0x008ac9d6cea723C5af947A42db2cA9719164B68c":{"balance":"10000000000000000000000"},"0x008320D31E9feC0246E4ee4Bbf28F15e3b9D1139":{"balance":"10000000000000000000000"},"0x00Ce65AF07b46696c2e4794A65f865d1Ae1141b7":{"balance":"10000000000000000000000"},"0x0097312Dc19cD5FDE27448C365d2cb8E174FD630":{"balance":"10000000000000000000000"},"0x00d894f19Bb120E078f5f2434A2eCDf5Fd2F89F5":{"balance":"10000000000000000000000"},"0x00c2D259A17D473Eb03400374696802FDc594Eed":{"balance":"10000000000000000000000"},"0x00b27E4200a01820cc98c0C7dF6a6dD43f69df06":{"balance":"10000000000000000000000"},"0x009947af362C1Ae4699fA4Bb23563A8E32E4A757":{"balance":"10000000000000000000000"},"0x00b03F0bc63438f96aAd7dD958EA9890e87Ae440":{"balance":"10000000000000000000000"},"0x00f05E3e6B6CD4aF641848BCb0d50b2372ec2e4D":{"balance":"10000000000000000000000"},"0x00B4E172ADc62e70568bb226C117Bf1c349B3407":{"balance":"10000000000000000000000"},"0x00e9cCbaF67E7881dA7E99AC3E02bA4aEE09033b":{"balance":"10000000000000000000000"},"0x00A20837b8C
```

Comment:
This looks like a qi address, but this balance is greater than the max denomination. Also, where is the utxo hash and index?

Reply:
yeah you are right, the allocs were hidden inside a different commit, now its fixed and can mine